src/components/expression/expression-renderer.js

```diff
@@ -137,36 +54,103 @@ genotet.ExpressionRenderer = function(container, data) {
   this.heatmapHeight_ = 0;
 
   /**
-   * Margin of the gene profile.
-   * @private {number}
+   * Margins of the gene profile.
+   * @private {!Object}
```

Comment:
Okay looks like markdown will eat angular bracketed elements...
```
@private {!Object<number>}
```
is shown as @private {!Object} outside code block.